setmyenv/startup.sh

```diff
@@ -0,0 +1,137 @@
+conda_url='https://repo.continuum.io/archive/Anaconda3-5.0.1-Linux-x86_64.sh'
```

Comment:
This points to a pretty specific version of anaconda doesn't it... maybe we should find the permanent URL for the newest anaconda?...  Newer anacondas appear in the archive (e.g. from december 2018) https://repo.anaconda.com/archive/

anyway I think the .io URL might not work forever (see: https://github.com/conda/conda/issues/6886) ..